_specs/1.0/index.md

```diff
@@ -277,10 +277,58 @@ A `Collection` **MAY** have the following:
 }
 ```
 
+## <a href="#document-entry-point" id="document-entry-point" class="headerlink"></a> Entry Point
 
-## <a href="#document-components-error-object" id="document-components-error-object" class="headerlink"></a> Error Object
+A `EntryPoint` is a type of [node](#document-components-node) used to represent a resource that clients can use to get more information about an API and provide [links](#document-components-link-collection) to traverse.
 
-Processing errors can be handled by returning a [node object](#document-components-node-object) of `Error` to consumers. In addition, an appropriate HTTP status code, a human readable `code` must be returned.
+It **MUST** be the top most JSON object and **MUST NOT** be nested.
+
+A `EntryPoint` **MUST** have the following:
+
+* `@id`: Represents a valid [URI](#conventions-uri).
+* `@type`: Have a value of `EntryPoint`.
+* `@links`: Repesents a [link](#document-components-link-collection) with navigational _links_ to other APIs and additional resources.
+    * `documentation`: Represents a [link value](#document-components-link-value) navigating to the documentation page.
+    * `support`: Represents a [link value](#document-components-link-value) navigating to the support page.
+
+* `name`: Represents the name of the API as string.
+
+A `EntryPoint` **MAY** have the following:
+
+* `description`: Represents a description of the API as string.
+* `version`: Represents the current version of the API following [SEMVER](https://semver.org) as string. 
```

Comment:
I don't think we're using semver.org strictly.  It specifics it all must be X.Y.Z, where we are more open to something like 'v1'  Maybe we can point people at the url design section instead?

Reply:
Ah yes i agree, i'll make that change